sunpy/time/timerange.py

```diff
@@ -201,7 +195,8 @@ def __eq__(self, other):
         result : `bool`
         """
         if isinstance(other, TimeRange):
-            return (self.start == other.start) and (self.end == other.end)
+            return _is_time_equal(
```

Comment:
This is a work around right? if so should have a comment

Reply:
Yeah. I'll add it.

Reply:
I added the comment to the doc string of the function.